packages/xod-project/src/project.js

```diff
@@ -548,6 +549,75 @@ export const validateLinkPins = def(
   }
 );
 
+export const getInvalidBoundNodePins = def(
+  'getInvalidBoundNodePins :: Project -> Patch -> Node -> StrMap (Either Error a)',
```

Comment:
Can’t easily understand what does the function return. A comment would help a lot (we document exported funcs, yep? :smiley: ).

In particular, what’s hidden behind `a`?

Reply:
Done